### ISSUE TITLE
Gives virus in cache folder

### ISSUE THREAD

**Issue by Viper3881** [WowUp/WowUp#1420]
![Screenshot 2023-02-28 185055](https://user-images.githubusercontent.com/113194527/222016762-e66b2655-2ef1-4b00-8d95-0933e5c3ad88.png)

**Comment by linaori** [comment]
Did this happen after downloading a specific addon?

**Comment by Viper3881** [comment]
I wish I knew what addon, but I have not changed specific addons for months. I only updated them when needed. Nothing popped up when I installed the updates at any single point in time.

**Comment by ZeShmoutt** [comment]
This just happened to me with the latest stable version of [Details](https://www.curseforge.com/wow/addons/details), using WowUP 2.20.0 CF.

**Comment by linaori** [comment]
@ZeShmoutt most likely a false positive

**Comment by ZeShmoutt** [comment]
Yeah,  I'm currently doing a full scan with something else than Windows Defender just to be sure. Still a bit weird that it's this specific addon and nothing else.

**Comment by linaori** [comment]
It happens occasionally, DBM also gets hit by this like once or twice a year
